### DISCUSSION TITLE
What AI coding assistant are you currently using with LiteLLM?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [BerriAI/litellm#6008] [Ideas]
I am wondering what AI coding assistants (IntelliJ IDEA & VSCode plugins) do everyone use with LiteLLM today?
Thanks! 😃

**Comment by hongbo-miao** [comment]
Personally I succeed

- Continue (Can be used in Visual Studio Code & IntelliJ IDEA)
  - My config
    ```json
    {
      "title": "claude-3-5-sonnet",
      "provider": "openai",
      "apiBase": "https://litellm.example.com",
      "apiKey": "anything",
      "model": "claude-3-5-sonnet"
    },
    ```
- Claude Dev (Can be used in Visual Studio Code)

**Reply by ricardosilva86** [reply]
I've read the reviews for continue plugin in JetBrains IDEs and 90% says it freezes the IDE every 1h, how is your experience with it?
